### DISCUSSION TITLE
NPM - Upload cartridge - Demo instance

### DISCUSSION THREAD

**Discussion by amitkathpal123** [SalesforceCommerceCloud/sfcc-ci#417] [Team Posts]
Does "npm run uploadCartridge" does not work for Demo instance. Probably because Demo can be now logged-in with Account manager username/password. Is the correct?

**Comment by jorgehernandezSF** [comment]
You can use your AM username and password in the dw.json configuration --
it should work just like a local user's credentials work.
Let us know if this works.

On Thu, Jun 27, 2019 at 3:06 PM amitkathpal123 <notifications@github.com>
wrote:

> Does "npm run uploadCartridge" does not work for Demo instance. Probably
> because Demo can be now logged-in with Account manager username/password.
> Is the correct?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SalesforceCommerceCloud/teams/community/discussions/65?email_source=notifications&email_token=AJZO4ZVYGWTSJKJ3O32SKETP4UFRVA5CNFSM4H37JO52YY3PNVWWK3TUL52HS4DFVZCGS43DOVZXG2LPNZIG643UVJRW63LNMVXHIX3JMTHAAAIXFQ>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJZO4ZSM2OZU3I55UH4TX73P4UFRVANCNFSM4H37JO5Q>
> .
>
-- 
Jorge Hernandez
Director, Global Cloud Alliances
mobile: 617-957-9110
email: jorge.hernandez@salesforce.com

**Comment by amitkathpal123** [comment]
I am always getting an error:
[10:31:49] Error: no auth mechanism defined
exec: [10:31:49] Error: no auth mechanism defined

This is not specific to Demo instance but coming for another sandbox as well.

**Comment by pravngaur** [comment]
Hello Amit,

1) Do you have dw.json defined.
2) are you able to upload the code through prophet debugger using the same
creds?

**Comment by amitkathpal123** [comment]
Thanks for helping.

I found the problem. It was with dw.json. File was present but somehow it was not formatted correctly due to conversion from .txt to .json.

**Comment by pravngaur** [comment]
Glad to hear that. Thanks
